README.md

```diff
@@ -80,11 +70,13 @@ See this [blog post](https://rossta.net/blog/how-to-specify-local-ruby-gems-in-y
 
 ### Testing
 
-To run tests in a Docker container, you can run:
+We have a dedicated `.env` file for test enviroment, which can be found at `.env.test.local`, which with our `dotenv` gem is loaded
+as a first priority in test environment. We mock values only when needed.
+
+In Github Actions, we run the tests with the local environment because setting up the containers and running them there is more expensive in terms of time. You can run the tests in the containers, if you don't want to get set up locally with:
 
 ```bash
-$ docker compose exec -it app bash
-$ RAILS_ENV=test rspec
+$ docker compose exec app bundle exec rspec
```

Comment:
I still have to set the env variable RAILS_ENV to test to make the test run. So I think we should replace this line with:

`$ docker compose exec -e RAILS_ENV=test app bundle exec rspec`